ownCloud/Viewer/ImageDisplayViewController.swift

```diff
@@ -38,12 +54,59 @@ class ImageDisplayViewController : DisplayViewController {
 		}
 	}
 
+	// MARK: - Frame changes
 	override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
 		super.viewWillTransition(to: size, with: coordinator)
 		self.scrollView?.updateScaleForRotation(size: size)
 	}
+
+	// MARK: - Gesture recognizers actions.
+	@objc func tapToZoom() {
+		guard let scrollView = scrollView else {
```

Comment:
@pablocarmu I think you can assume that scrollView is always accessible and just force unwrap it